pyproject.toml

```diff
@@ -35,5 +38,20 @@ condax = "condax.cli:cli"
 [tool.hatch.version]
 source = "vcs"
 
+[tool.hatch.envs.default]
+dependencies = [
```

Comment:
Did you consider here instead using [groups of optional dependencies](https://hatch.pypa.io/latest/config/environment/overview/#features)? That would allow you to define groups which could be combined.

Reply:
typically when creating the development environment I'm not concerned about installing some extra dependencies that I'm not going to specifically use. e.g. if i'm changing the implementation of a function but i wont require to change any docs, i dont mind if my development environment contains all the dev dependencies including the doc generation tools despite the fact that im not going to use them. this makes life easier for devs because they don't need to start considering which groups of dev deps they'll want in their env. might as well always download all of them

Reply:
I didn't mean to suggest splitting up the dev deps into categories...

Currently you're installing the dev dependencies into the default environment. Probably this is fine, but I was thinking that having them in a separate group may make it easier in case someone wants to locally run `condax` in an environment without the dev dependencies. Or maybe one wants to test an alternative set of dev dependencies, then it becomes cleaner to simply point to a group.

I don't really have any workflow for using the Hatch CLI, so I honestly have no idea what is best practice here. I'm mainly just curious.